hack/tools.mk

```
@@ -136,3 +138,9 @@ $(YAML2JSON): go.mod
 $(YQ): $(call tool_version_file,$(YQ),$(YQ_VERSION))
 	curl -L -o $(YQ) https://github.com/mikefarah/yq/releases/download/$(YQ_VERSION)/yq_$(shell uname -s | tr '[:upper:]' '[:lower:]')_$(shell uname -m | sed 's/x86_64/amd64/')
 	chmod +x $(YQ)
+
+$(COMPONENT_CLI): go.mod
```

```suggestion
$(COMPONENT_CLI): $(call tool_version_file,$(COMPONENT_CLI),$(COMPONENT_CLI_VERSION))
```

See https://github.com/gardener/gardener/blob/a264b0a08bada2df6a1b7169c1c061abf5ca3400/hack/tools.mk#L58-L66

---

thanks!